### DISCUSSION TITLE
can it be used on window 10?

### DISCUSSION THREAD

**Discussion by shiweizheng** [NVlabs/curobo#193] [Software Q&A] [answered]
I try to install and compile curobo on Windows 10. I followed the Installaion in https://curobo.org/source/getting_started/1_install_instructions.html. While I run "pip install -e . --no-build-isolation", I get error:
![20240321-181804](https://github.com/NVlabs/curobo/assets/19202332/ef504a97-aa54-4a36-a0e1-d12ae1270755)

**Comment by bhavindharia** [comment]
I read in one of the thread that windows is not supported yet. My question is that is there a plan to make it supported for windows? at least for the python api (No isaac).

**Comment by balakumar-s** [comment]
There is no plan to support windows. However, some users were able to get it running in windows: https://github.com/NVlabs/curobo/discussions/37#discussioncomment-7571229

**Reply by bhavindharia** [reply, marked as answer]
Thanks for the reply @balakumar-s. Someone has made a branch for windows support. I was able to build it in my windows 11 device directly without wsl2.  For people in need for installing the great curobo, please checkout this branch:
(https://github.com/wawatt/curobo_for_windows)

If in future you want to add support for windows. This will be a good start.
